### PR TITLE
Fix for running the job manually from the Admin panel.

### PR DESCRIPTION
**Pull request by littlejon**
When running the scheduled task via `System Settings -> Scheduled Jobs` via `Execute Now` the default script timeout of 300 seconds isn't long enough to complete the job with a large contact list.

This change extends the scripts capable runnning time to 1800 seconds.

### CODE REVIEW

**Comment by agileware-justin** [comment]
@littlejon thanks for checking this extension out and providing a PR. I hope you find the extension useful.

The scheduled job included with this extension is intended to run in the background and you would definitely want to do that if your CRM has a large number of contacts.

Ideally, there would be an easy way to queue long running jobs like this one. So that when the PHP timeout is reached, the job restarts where it left off when the job executes again. Which is a reasonable expectation for modern systems.

There has been a fair amount of discussion about how this might be implemented over here, https://lab.civicrm.org/dev/core/-/issues/1304 - Define more usable queue DX for multithreaded background work

So in short, implementing a queue model would be ideal.

**Comment by agileware-justin** [comment]
Marking this as closed, since this can be set for your website or CiviCRM settings and is not specific to this extension.

Thanks anyway and have a great day!
